compiler-rt/lib/plsan/plsan.cpp

```diff
@@ -56,8 +55,12 @@ extern "C" LazyCheckInfo *__plsan_free_local_variable(void **arr_start_addr,
   LazyCheckInfo *lazy_check_info =
       (LazyCheckInfo *)__sanitizer::InternalAlloc(sizeof(LazyCheckInfo));
   lazy_check_info->RefCountZeroAddrs = ref_count_zero_addrs;
-  lazy_check_info->ProgramCounterAddr = __builtin_return_address(0);
-  return lazy_check_info;
+  if (is_return) {
+    __sanitizer::InternalFree(lazy_check_info);
```

Comment:
So this commit does more than it describes:
- remove unused field ProgramCountAddr (because it a stack trace always contains the counter)
- Fix a memory leak when lazy check does not free lazy check info

Reply:
Plsan don't need program counter whne leaks is occuered.
when refCnt is 0,  get stack trace id using refsetLeakedLoc() instead this code.